client_tls.go

```diff
@@ -45,11 +45,14 @@ func appendCert(path string, pool *x509.CertPool) {
 	log.Debugf("Appended cert from '%s'", path)
 }
 
-// NewCertPool loads PEM certificates from given path and returns them in a way that is usable at TLS() as RootCAs.
+// NewCertPool initiates system's cert pool then loads PEM certificates from given path and returns them in a way that is usable at TLS() as RootCAs.
 // If path is a directory then scans for files recursively. If path is not set then defaults to /etc.
 // File name should match *.crt or *.pem.
 func NewCertPool(path string) *x509.CertPool {
```

Comment:
I think it is better having a bool parameter telling whether to add the cert to the system certs. And have that from `TLSRootCerts`.